roles/k8s_best_practices_certsuite/tasks/tests.yml

```diff
@@ -11,6 +11,16 @@
     state: directory
     mode: '0755'
 
+- name: Verify if the CWE Hydra API key file exists
```

Comment:
This should only be run if certsuite version is higher or equal than v5.4.2 or if using HEAD

Reply:
good point